qiskit_dynamics/__init__.py

```diff
@@ -19,11 +19,12 @@
 
 .. currentmodule:: qiskit_dynamics
 
-qiskit extension module for solving ordinary differential equations.
+Qiskit extension module for simulating quantum dynamics.
```

Comment:
Yeah that's a good question... I'm not sure what makes the most sense structure-wise. One option I was thinking: 

- Move both `solve.py` and `solver_classes.py` into the `solvers` directory, and put the API docs for these into `solvers/__init__.py`. 
- We can still keep these functions/classes as root-level imports.

I like this option from a code/functionality compartmentalization perspective.

I'm not totally sure what:
> since they aren't in the default import path anymore

is referring to.